EventGenerator/src/Pileup_module.cc

```diff
@@ -66,6 +70,14 @@ namespace mu2e {
 
     //----------------------------------------------------------------
   private:
+    Int_t nEv;
```

Comment:
Please remove the histogramming (just revert the module); that isn't needed now you made the analyzer.

Reply:
will do